### ISSUE TITLE
Explorer does not start correctly

### ISSUE THREAD

**Issue by smallestant** [digital-asset/daml-on-fabric#12]
When starting Explorer service, got error:

```
REST: Blocks reader: Caught Exception:
com.daml.FabricContextException: response {
  status: 500
  message: "Unknown action, check the first argument, must be one of \'RawWrite\', \'RawRead\'. But got: GetChainInfo"
}
payload: "\n S\nmK\372\235\016\341=J3s\371B\242\306\233\313a\300\233z\301\000?\016K|\230\203\266\005\022\360\001\nm\022?\n\n_lifecycle\0221\n/\n)namespaces/fields/daml_on_fabric/Sequence\022\002\b\003\022*\n\016daml_on_fabric\022\030\n\026\n\020\000\364\217\277\277initialized\022\002\b\004\032j\b\364\003\022eUnknown action, check the first argument, must be one of \'RawWrite\', \'RawRead\'. But got: GetChainInfo\"\023\022\016daml_on_fabric\032\0011"

	at com.daml.FabricContext.querySystemChaincode(FabricContext.java:609)
	at com.daml.ExplorerService$BlocksReader.run(ExplorerService.java:514)
```

**Comment by smallestant** [comment]
Correction suggestion:

In FabricContext.java
Line 602:

from         req.setChaincodeName(ccName);

to             req.setChaincodeName(cc);

**Comment by dasormeter** [comment]
closed with #13
